### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by luca-libonati** [fattureincloud/api#349] [Q&A] [answered]
Buongiorno,
ho appena aperto un account per fare dei test, seguendo questi passaggi:

- ho creato un'azienda di test
- sono andato nell'area Sviluppatore e attivato Oauth2
- ho installato composer seguendo quanto indicato qui https://developers.fattureincloud.it/docs/quickstarts/php-sdk-quickstart/
- ho creato il file oauth.php https://developers.fattureincloud.it/docs/quickstarts/php-sdk-quickstart/#2%EF%B8%8F%E2%83%A3-step-two-set-up-the-oauth-access-token-retrieval

Ricevo errore 500 (applicativo gira su PHP 8.1)

Se provo a mettere in pagina anche solo quanto sotto, ottengo un errore 500.
<?
ini_set('display_errors', 1); ini_set('display_startup_errors', 1); error_reporting(E_ALL);
require "../vendor/autoload.php"
use FattureInCloud\OAuth2\OAuth2AuthorizationCodeManager;
?>

Manca qualcosa che mi sono perso?

Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @luca-libonati, con così poche informazioni è difficile capire cosa stia causando il problema, quello che sembrerebbe però é che l'errore 500 venga dal tuo server e non dalla nostra api, solitamente è dovuto a qualche errore di import o di codice, riuscirebbe a debuggare in qualche modo? (anche solamente mettendo degli echo sparsi per il codice per capire dove accade specificamente dove viene tirata l'eccezione).

[Qui](https://github.com/fattureincloud/fattureincloud-quickstarts/tree/master/sdk/php) può trovare un semplice esempio completo e funzionante di autenticazione e chiamata di creazione fornitore.

Grazie,
Emanuele.

**Reply by luca-libonati** [reply]
Ciao Emanuele,
mio errore nel codice. 
Sorry,
L.
